packages/deploy/hardhat.config.ts

```diff
@@ -312,6 +312,16 @@ const compilers = [
   },
 }));
 
+compilers.push({
+  version: '0.8.15',
```

Comment:
Do we need this since you deleted 0.8.15 above?

Reply:
yes, it's adding there to have custom "run"